vision/mnist/conv.jl

```diff
@@ -68,17 +74,21 @@ model = gpu(model)
 # Make sure our model is nicely precompiled before starting our training loop
 model(train_set[1][1])
 
+# We augment `x` a little bit here, adding in random noise. 
+augment(x) = x .+ gpu(0.1f0*randn(eltype(x), size(x)))
```

Comment:
we should also remove this kind of data augmentation, it's not common at all, I don't see why it should get into one of the basic scripts that people look at as a template for their code.

Reply:
A message about basic augmentation would be nice, just to show how to get it working